workflow/scripts/gen_new_features.py

```diff
@@ -62,7 +61,7 @@ def determine_num_candidate_enh_gene(pred_df, results_dir):
 
     df["NumCandidateEnhGene"] = df["NumCandidateEnhGene"].astype("int")
     df[["name", "TargetGene", "NumCandidateEnhGene"]].to_csv(
-        os.path.join(results_dir, "NumCandidateEnhGene.tsv"),
+        os.path.join(results_dir, "NumCandidateEnhGene.txt"),
```

Comment:
any reason for changing the name to a txt file?

Reply:
They needed to be .txt files to be compatible with the other methods :)